.golangci.yaml

```diff
@@ -0,0 +1,42 @@
+linters-settings:
+  misspell:
+    locale: US
+linters:
+  disable-all: true
+  enable:
```

Comment:
The list seems quite extensive and I'm not sure if they all make sense.
How about we follow kubernetes config: either https://github.com/kubernetes/kubernetes/blob/master/hack/golangci.yaml or https://github.com/kubernetes/kubernetes/blob/master/hack/golangci-strict.yaml

Reply:
Thank you. This is a great pointer. I adopted style and metalinters from k8s. I removed `ginkgolinter` and `logcheck` as they are specific for their repository. Instead, I added `misspell` (checks for common English misspelled words in comments) and `gosimple` (one of the default linters that specializes in simplifying code). See mode details about each linter [here](https://golangci-lint.run/usage/linters/).